source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -1,29 +1,69 @@
 #pragma once
 
-#include "pch.h"
 #include "Enums.h"
 #include "json/json.h"
 
 namespace AdaptiveSharedNamespace
 {
-    struct FontSizesConfig
+    class FontSizesConfig
     {
-        unsigned int smallFontSize = 10;
-        unsigned int defaultFontSize = 12;
-        unsigned int mediumFontSize = 14;
-        unsigned int largeFontSize = 17;
-        unsigned int extraLargeFontSize = 20;
+    public:
+        FontSizesConfig() = default;
+        FontSizesConfig(unsigned int s, unsigned int d, unsigned int m, unsigned int l, unsigned int xl) :
+            smallFontSize(s), defaultFontSize(d), mediumFontSize(m), largeFontSize(l), extraLargeFontSize(xl)
+        {
+        }
 
         static FontSizesConfig Deserialize(const Json::Value& json, const FontSizesConfig& defaultValue);
+        static unsigned int GetDefaultFontSize(TextSize size);
+        unsigned int GetFontSize(TextSize size) const;
+        void SetFontSize(TextSize size, unsigned int value);
+
+    private:
+        // UINT_MAX used to check if value was defined
+        unsigned int smallFontSize = UINT_MAX;
```

Comment:
>smallFontSize [](start = 21, length = 13)

Now that these are classes, let's name these as member variables (m_smallFontSize, for example)